news-TS/src/components/controller/controller.ts

```diff
@@ -0,0 +1,118 @@
+import AppLoader from './appLoader';
+import { drawContentFunc, IAppController, apiOptions } from '../../types/index';
+
+class AppController extends AppLoader implements IAppController {
+    getSources(callback: drawContentFunc): void {
+        super.getResp(
+            {
+                endpoint: 'sources',
+            },
+            callback
+        );
+    }
+
+    getTopHeadlines(callback: drawContentFunc): void {
+        const permitedCountries: string[] = [
```

Comment:
There is no need to set types everywhere because typescript can infer them, so 
```
const permitedCountries = ['ae', ...];
```
will be enough